src/util/comparison.coffee

```diff
@@ -55,6 +55,12 @@ module.exports.equivalent = equivalent = (a, b) ->
       return compareEveryItemInArrays(a, b, equivalent)
     when '[object Object]'
       return compareObjects(a, b, equivalent)
+    when '[object String]'
+      # Make sure b is also a string
+      if bClass == '[object String]'
+        # String equivalence is case-insensitive
+        a = a.toLowerCase()
+        b = b.toLowerCase()
```

Comment:
The spec notes that equivalence should be case and locale insensitive, but I think this will only achieve the first part. Looks like you should be able to use the js feature "localeCompare".

Reply:
Isn't _not_ using localeCompare locale-insensitive behavior?

Reply:
Hmm I dont think so. I think `define LocaleInsensitiveStrings: 'baréóa' ~ 'bArEoÄ'` should resolve to true.
`"baréóa".localeCompare("bArEoÄ", 'en', {sensitivity: 'base'})` is 0 (same), but `"baréóa".toLowerCase() == "bArEoÄ".toLowerCase()` is false